corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name
```

Comment:
- [ ] the `correct_name` method should accept `name` as an argument.

`def correct_name(name)`